benchmark/speed/benchmark_rcf.jl

```diff
@@ -1,80 +0,0 @@
-#!/usr/bin/env julia
```

Comment:
`benchmark_quarter_degree.jl` is a better benchmark, so I'm getting rid of this one to avoid confusion. There aren't enough sources in this rcf for each thread to have enough work.